### PR TITLE
CRUD Generator Tutorial

### PR DESCRIPTION
**Pull request by thomasdax98**

### CODE REVIEW

File: demo/admin/crud-generator-config.ts [hunk 1]
```diff
@@ -8,4 +8,12 @@ export default [
         target: "src/news/generated",
         entityName: "News",
     },
+    {
+        target: "src/books/generated",
+        entityName: "Book",
+        rootBlocks: {
+            coverImage: { name: "DamImageBlock", import: "@comet/cms-admin" },
+            link: { name: "LinkBlock", import: "../../common/blocks/LinkBlock" },
```
Comment (nsams) [inline demo/admin/crud-generator-config.ts:16]: custom rootBlocks should not be neccasary for common cases and should be avoided

Comment (thomasdax98) [reply demo/admin/crud-generator-config.ts:16]: Oh okay. I didn't get that. So it's not necessary for these two examples

Comment (thomasdax98) [reply demo/admin/crud-generator-config.ts]: I added a notice to the config

File: demo/admin/crud-generator-config.ts [hunk 2]
```diff
@@ -8,4 +8,18 @@ export default [
...
         entityName: "News",
     },
+    {
+        target: "src/books/generated",
+        entityName: "Book",
+        // if your entity expects block JSON for some props, you might need to explicitly define rootBlocks
+        // that's not necessary if the used block is defined
+        //   - in src/common/blocks and conforms to our naming conventions
+        //   - in a blocks folder right next to the generated folder and conforms to our naming conventions
+        //   - in @comet/cms-admin or @comet/blocks-admin
```
Comment (thomasdax98) [inline demo/admin/crud-generator-config.ts:18]: @nsams Is that correct now?

Comment (nsams) [reply demo/admin/crud-generator-config.ts:18]: yes; would write this into the docs though

Comment (thomasdax98) [reply demo/admin/crud-generator-config.ts:18]: Sure, still this PR should serve as a simple tutorial on how to use the admin generator. So it's good to have it here too.

**Pull request closed without merging**